api/GetEmbedInfo/embedConfigService.js

```diff
@@ -23,9 +23,13 @@ async function getEmbedInfo () {
     };
   } catch (err) {
     return {
-      status: err.status,
-      error: `Error while retrieving report embed details\r\n${err.statusText}\r\nRequestId: \n` +
-        `${err.headers.get('requestid')}`
+      error: {
+        status: err.status,
+        statusText: err.statusText,
+        powerBIErrorInfo: err.headers.get('x-powerbi-error-info'),
+        description: `Error while retrieving report embed details\r\n${err.statusText}\r\nRequestId: \n` +
```

Comment:
Is it something we want to translate?

Reply:
This may be used later as a console error, but it isn't shown in the User Interface right now, so I think we don't want to translate it :)

Reply:
Then LGTM :)